docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:9.0.0
+    container_name: mysql
+    ports:
+      - "3307:3307"
+    networks:
+      - rihal
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: usersystem
+      MYSQL_PASSWORD: root 
+      MYSQL_TCP_PORT: 3307
+    volumes:
+      - cont_db_data:/var/lib/mysql
+    restart: always 
+  app:
+    build: .
+    container_name: java_app
+    ports:
+      - "8080:8080"
+    networks:
+      - rihal
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysql:3307/usersystem?useSSL=false&serverTimezone=UTC&createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true
+      SPRING_DATASOURCE_USERNAME: root
+      SPRING_DATASOURCE_PASSWORD: root
```

Comment:
We should never push the password on GitHub.
Using the .env file can help solve this issues.